@here/harp-examples/src/datasource_satellitetile_globe.ts

```diff
@@ -35,8 +35,7 @@ export namespace SatelliteTileDataSourceGlobeExample {
         const map = new MapView({
             canvas,
             theme: "resources/berlin_tilezen_base_globe.json",
-            projection: sphereProjection,
-            tileCacheSize: 400
+            projection: sphereProjection
         });
         map.setCacheSize(100, 100);
 
```

Comment:
what about the call to `setCacheSize`? can we find a way to remove it?

Reply:
Didn't realize, thanks for pointing it out!